ledger/apptxn_test.go

```diff
@@ -62,19 +62,24 @@ func TestPayAction(t *testing.T) {
 
 		// We're going to test some payout effects here too, so that we have an inner transaction example.
 		proposer := basics.Address{0x01, 0x02, 0x03}
+		stateProofPK := merklesignature.Commitment{0x03}
+		if ver < 31 { // no state proof support
```

Comment:
these consensus version numbers getting spread across the codebase so I wonder if we need to name them